.eleventy.js

```diff
@@ -225,6 +225,8 @@ export default function (eleventyConfig) {
   eleventyConfig.addPlugin(EleventyHtmlBasePlugin);
 
   eleventyConfig.addPassthroughCopy("site/assets");
+  eleventyConfig.addPassthroughCopy("additional-data/timeline-number.json");
+  eleventyConfig.addPassthroughCopy("additional-data/timeline-durations.json");
```

Comment:
I've sort of designed the `additional-data` folder to contain web-features data that's not in the web-features repo. Vendor positions, mdn docs, and origin trials so far. Things that should, ideally, be hosted by those external parties instead, but which don't really have a home right now.

The timeline data is different from that. It's computed from web-features data. 
I'd prefer to keep it under the `assets` folder instead, which is used for images, css, js (public client-side stuff), and leave the `additional-data` folder out of the public website.